### PR TITLE
Project ARES: AI Core Makeover

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
With fantastic sprites from Zenith, and some additions by myself, the AI core is getting a visual overhaul!

# Explain why it's good for the game

Adds to the feel of the AI core, making it much more like the one it is supposed to be imitating.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image
image
Lights in the above have been removed in favour of the glowing floor tiles.
image

</details>

# Changelog

:cl:
imageadd: Added AI Core themed walls, windows and floors from Zenith.
imageadd: Added AI Core themed blast doors from Zenith.
imageadd: Added a few additional AI Core themed floors from myself.
maptweak: Remapped the AI core to use the new walls/floors.
add: Added a subtype of AI Core floor that glows orange for thematic lighting.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: maps/map_files/USS_Almayer/USS_Almayer.dmm
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:1]: E/W computer consoles in the room need a bit more light near them, too much of a dark spot currently.
bilde

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: maps/map_files/USS_Almayer/USS_Almayer.dmm
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:1]: mapping lgtm

**Review by Nanu308 — APPROVED** [review]

**Review by sleepynecrons — APPROVED** [review]
sprites approved

File: code/game/machinery/doors/poddoor.dm [hunk 1]
```diff
@@ -283,22 +283,40 @@
 		/obj/structure/machinery/door/airlock,
 	)
 
+/obj/structure/machinery/door/poddoor/almayer/Initialize()
+	. = ..()
+	addtimer(CALLBACK(src, TYPE_PROC_REF(/atom, relativewall_neighbours)), 10)
```
Comment (Drulikar) [inline code/game/machinery/doors/poddoor.dm:288]: Can we not use LateInitialize instead?

**Review by Drulikar — CHANGES_REQUESTED** [review]
Otherwise looks fine.

File: code/game/machinery/doors/poddoor.dm (hunk 1 above)
Comment (realforest2001) [reply code/game/machinery/doors/poddoor.dm:288]: I don't actually know how this got here, I don't recall putting it in.

**Review by Drulikar — APPROVED** [review]
image

Screenshotted again since the windows seemed to be corrected since the original images.

**Pull request merged**